packages/components/psammead-most-read/src/testHelpers/fixtureData.js

```diff
@@ -0,0 +1,15 @@
+const LTRLink = {
+  language: 'news',
+  title: 'John Lewis staff bonus cut again as profits fall',
+  href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+};
+
+const RTLLink = {
+  language: 'arabic',
```

Comment:
```suggestion
  service: 'persian',
```
This is to match the story title below.